x-pack/plugins/apm/public/plugin.ts

```diff
@@ -159,6 +162,9 @@ export class ApmPlugin implements Plugin<ApmPluginSetup, ApmPluginStart> {
       order: 8500,
       euiIconType: 'logoObservability',
       category: DEFAULT_APP_CATEGORIES.observability,
+      meta: {
+        keywords: ['UX'],
```

Comment:
Added for testing purposes. The intent is to let plugin developers add their own keywords.
`UX` inspired by https://github.com/elastic/kibana/issues/76778#issuecomment-704606826